dev/breeze/tests/test_pytest_args_for_test_types.py

```diff
@@ -105,7 +106,14 @@
         (
             GroupOfTests.PROVIDERS,
             "All-Quarantined",
-            ["providers/airbyte/tests", "providers/tests", "-m", "quarantined", "--include-quarantined"],
+            [
+                "providers/airbyte/tests",
```

Comment:
FYI We should likely make this automatically calculated to not having to remmber about those. I might add it as a separate PR

Reply:
Yeah, that current approach does not scale. Just also changed one-by-one.
I'd also propose to separate this to another PR.

Reply:
yep.